deployment/README.md

```diff
@@ -114,6 +114,8 @@ The deployer must run under a service account defined as follows:
     $ oc policy add-role-to-user daemonset-admin --serviceaccount logging-deployer
     $ oadm policy add-cluster-role-to-user oauth-editor \
               system:serviceaccount:logging:logging-deployer
+    $ oadm policy add-cluster-role-to-user daemonset-scheduler \
```

Comment:
I tried to add this first as a normal role but it didn't let me list and patch the nodes in the cluster as it would want to